packages/central-server/src/apiV2/userFavouriteDashboardItem/POSTUpdateUserFavouriteDashboardItem.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { respond } from '@tupaia/utils';
+import { assertAnyPermissions } from '../../permissions';
+import { CRUDHandler } from '../CRUDHandler';
+
+/**
+ * Handles POST endpoints:
+ * - /userFavouriteDashboardItem
+ */
+
+export class POSTUpdateUserFavouriteDashboardItem extends CRUDHandler {
+  async assertUserHasAccess() {
+    const assertUserIsLoggedIn = () => {
+      const { userId } = this.req;
+      if (!userId) {
+        throw new Error('User should be logged in');
+      }
+    };
+
+    return this.assertPermissions(assertAnyPermissions([assertUserIsLoggedIn]));
+  }
+
+  async handleRequest() {
+    const { models, body, userId } = this.req;
+    const { dashboardItemCode, changeType } = body;
```

Comment:
Instead of `changeType` being `create` or `delete` here, wanna instead stick with the `favourite` and `unfavourite` terms? 

So the request looks like:
```
POST /userFavouriteDashboardItem
{
  dashboardItemCode: 'LESMIS_viz',
  state: 'favourite'
}

or

POST /userFavouriteDashboardItem
{
  dashboardItemCode: 'LESMIS_viz',
  state: 'unfavourite'
}
```

I like the idea of hiding the fact that we're creating/deleting individual `userFavouriteDashboardItem` records. That way the client can just think about it as favouriting and unfavouriting 👍